ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/PermissioningConfigurationBuilder.java

```diff
@@ -137,23 +134,15 @@ private static LocalPermissioningConfiguration loadAccountPermissioning(
   }
 
   /**
-   * This method allows support for both keys for now. Whitelist TOML keys will be removed in future
-   * (breaking change)
+   * This method allows support for allowlist keys only.
    *
    * @param tomlParseResult result of a prior toml parse
    * @param primaryKey key to fetch
-   * @param alternateKey alternate key to fetch
-   * @return In order: the array of the primaryKey if it exists, or the array of the alternateKey if
-   *     it exists, or null.
+   * @return The array of the primaryKey if it exists, or null.
    */
   private static TomlArray getAllowlistArray(
-      final TomlParseResult tomlParseResult, final String primaryKey, final String alternateKey) {
-    final TomlArray array = tomlParseResult.getArray(primaryKey);
-    if (array == null) {
-      return tomlParseResult.getArray(alternateKey);
-    } else {
-      return array;
-    }
+      final TomlParseResult tomlParseResult, final String primaryKey) {
```

Comment:
nit: should it just be called `key` now?

Reply:
renamed